http/grpc/server.go

```diff
@@ -0,0 +1,55 @@
+package grpc
```

Comment:
should we name this server.go file as handler.go and it is consistent with how other protocol sub packages have handlers of their own and plus this server.go file is not a server anyway. wdyt?

Reply:
Agreed!